docs/content/reference/ransomware.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Ransomware"
+date: 2021-06-23T18:13:59+05:30
+draft: true
+pre: '<i class="fas fa-lock"></i> '
+weight: 10
+---
+
+The Infection Monkey has the capability of simulating a ransomware attack on your network.
+All actions performed by the encryption routine are designed to be safe for production
+environments.
+
+To ensure minimum interference and easy recoverability, the ransomware simulation will encrypt
+files if the user specifies a directory that contains files that are safe to encrypt.
+If no directory is specified, no files will be encrypted.
```

Comment:
@mssalvatore is this important enough to be added to the description of directory input fields in config UI?!

Reply:
I feel it is. What if the user hasn't read the documentation?

Reply:
Within reason, I think a self-documenting UI is better than just relying on just documentation. I vote yes on adding this information to the UI. How do we do this now, a "description" field?

Reply:
Yes, the same way we're doing it for custom PBA commands.

Reply:
@VakarisZ @shreyamalviya I've added a task to #1239 to make sure we don't forget.